### PR TITLE
Allow read-only custom feature browsing

### PR DESCRIPTION
**Pull request by anthonysena**
Modifies the custom feature editor to allow for browsing concept sets when in read-only mode per #2532. These changes also enable the copy SQL to clipboard per #2812.

### CODE REVIEW

File: js/pages/characterizations/components/feature-analyses/fa-view-edit/fa-design.html [hunk 1]
```diff
@@ -100,35 +103,37 @@
...
-                    <div data-bind="if: $data.criteriaType === 'CriteriaGroup'">
-                        <criteria-group params="{ expression: { ConceptSets: $component.data().conceptSets }, group: $data.expression }"></criteria-group>
-                    </div>
-                    <div data-bind="if: $data.criteriaType === 'DemographicCriteria'">
-                        <demographic-criteria params="{ criteria: $data.expression }"></demographic-criteria>
-                    </div>
-                    <div data-bind="if: $data.criteriaType === 'WindowedCriteria'">
-                        <windowed-criteria params="{ expression: { ConceptSets: $component.data().conceptSets }, criteria: $data.expression, disableObservationPeriod: true, defaultObservationPeriod: true }"></windowed-criteria>
-                    </div>
+				<div data-bind="css: $component.canEdit() ? '' : 'feature-analysis-view-edit__content--disabled'">
```
Comment (anthonysena) [inline js/pages/characterizations/components/feature-analyses/fa-view-edit/fa-design.html:106]: This CSS class `feature-analysis-view-edit__content--disabled` is defined by the parent component and I was not sure how to best use it so for now it is hardcoded.

Comment (anthonysena) [reply js/pages/characterizations/components/feature-analyses/fa-view-edit/fa-design.html:106]: @alex-odysseus do you or anyone from your team provide a suggestion to handle this CSS approach?

Comment (anthonysena) [reply js/pages/characterizations/components/feature-analyses/fa-view-edit/fa-design.html:106]: Just to follow up on this item: I've attempted to edit the child .less sheets to include the `disabled` BEM modifier without much luck. I'd propose just hard-coding this now.

**Review by chrisknoll — APPROVED** [review]

File: js/components/multi-select.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ define(
 
       self.multiple = params.multiple;
       self.options = params.options;
+	  self.disable = (params.disable || false);
```
Comment (pieterlukasse) [inline js/components/multi-select.js:17]: should this be `self.disabled` to match `attr: disabled,` added to the .html above?

**Pull request merged**